crates/wasi-provider/Cargo.toml

```diff
@@ -24,18 +24,18 @@ anyhow = "1.0"
 async-trait = "0.1"
 backtrace = "0.3"
 kube = { version = "0.55", default-features = false }
-wasmtime = "0.25"
-wasmtime-wasi = "0.25"
-wasi-common = "0.25"
-wasi-cap-std-sync = "0.25"
+wasmtime = { version = "0.28", default-features = true }
+wasmtime-wasi = "0.28"
+wasi-common = "0.28"
+wasi-cap-std-sync = "0.28"
 cap-std = "0.13"
 tempfile = "3.1"
 serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 kubelet = { path = "../kubelet", version = "0.7", default-features = false, features = ["derive"] }
 krator = { version = "0.3", default-features = false, features = ["derive"] }
-wat = "1.0"
+wat = "1.0.38"
```

Comment:
Is this pin necessary or can we use anything from the 1.0 series? I know it will select higher, but I was just curious

Reply:
It is, Wasmtime fails to compile with anything lower than 1.36 I think?